paddlespeech/cls/exps/panns/deploy/predict.py

```diff
@@ -24,7 +24,7 @@
 # yapf: disable
 parser = argparse.ArgumentParser()
 parser.add_argument("--model_dir", type=str, required=True, default="./export", help="The directory to static model.")
-parser.add_argument('--device', choices=['cpu', 'gpu', 'xpu'], default="gpu", help="Select which device to train model, defaults to gpu.")
+parser.add_argument('--device', choices=['cpu', 'gpu', 'xpu', 'gcu'], default="gpu", help="Select which device to train model, defaults to gpu.")
```

Comment:
Can GCU be used locally to run?

Reply:
Yes.We locally try to run all the supported scenarios of paddlespeech on gcu.It can run well with an extra option '--device=gcu'